src/app/config/schema.ts

```diff
@@ -286,6 +286,12 @@ export const optionalVarsSchema: Schema<IOptionalVarsSchema> = {
       env: 'SEND_AUTH_OTP_RATE_LIMIT',
     },
   },
+  corsWhitelist: {
+    doc: 'Comma separated domain strings to enable CORS for',
```

Comment:
does `convict` parse comma-separated strings correctly as arrays? can't find it in the docs

Reply:
yea, it's a convict type we created in `src/app/config/schema.ts#19`, coerces the input (which is always string due to being an env var) into an array

```ts
convict.addFormat({
  name: 'string[]',
  validate: (val: string[]) => {
    if (!Array.isArray(val)) {
      throw new Error('must be of type Array')
    }
    if (val.some((i) => typeof i !== 'string')) {
      throw new Error('Elements must be of type string')
    }
  },
  coerce: (val: string): string[] => {
    return val.split(',')
  },
})
```